book/types_of_data.md

```diff
@@ -203,10 +203,6 @@ You can write binary as a literal using any of the `0x[...]`, `0b[...]`, or `0o[
 
 Incomplete bytes will be left-padded with zeros.
 
-## Structured data
```

Comment:
I'd rather see this left in or rewritten. A founding principle of nushell is working with structured data. If it's not defined anywhere it makes it more difficult to understand. We refer to structured data in discord all the time.